### DISCUSSION TITLE
Demo not working

### DISCUSSION THREAD

**Discussion by pakricard** [timschofield/webERP#240] [Problems / Bugs]
Hi all:
https://weberp.org/demo/index.php just returns error 500 after logging in as admin/kwamoja

Can't manage to pass it.

What webERP version is currently run on the demo?

Tried to check the SecurityTokens.php script as the newest version doesn't seem to work in my installation. Once the demo is working again, I'll try again.

**Comment by dalers** [comment]
[SecurityTokens.php](https://github.com/timschofield/webERP/blob/master/SecurityTokens.php) was last updated 2 months ago, so I would suspect the login problem is the side effect of some other change. I'll try going back through the commits to see when I could last edit user records in www_users.

**Comment by pakricard** [comment]
Now doesn't return error 500 but: "incorrect password The user/password combination is not a valid user of the system"
trying with admin / kwamoja
After this first error the dropdown for the company doesn't work anymore.

The v4.15.2 SecurityTokens.php error is unrelated to the login of the demo, but it shows some HTML code on the entry fields as in the attached file:
![error in SecurityTokens](https://github.com/user-attachments/assets/2f3644fb-f8ed-4d85-b51a-5537e3b36684)

**Comment by dalers** [comment]
This may not be related but I found the commit when the Edit/Delete links in WWW_Users.php stops being shown.

## Nov 13 commit 2

```
dale@whizzer:/usr/local/www/weberp $ sudo -u www git checkout c8bd38a9e8c85928c99174c93279b5c9527e9f56
Previous HEAD position was 24c679ae7 Correct default time in the ConvertSQLDateTime() function. Should be 00:00:00 as reported by Pak Ricard
HEAD is now at c8bd38a9e Images in manual were showing at 32px x 32px size. Correct the css to show at 50% width
dale@whizzer:/usr/local/www/weberp $
```
FAILS weberp_scc db: WWW_Users.php does NOT show Edit/Delete links for user records

**Comment by timschofield** [comment]
That's weird. I know for a fact I grepped for every occurrence of kwamoja.
I must have copied something back over at some point. The user/password
should be admin/weberp

I have changed this. The strange thing is lots of people have been logging
into it.

Not sure how this happened, but should be fixed.
Tim

On Sat, 16 Nov 2024 at 20:02, Dale Scott ***@***.***> wrote:

> Found the commit when the Edit/Delete links in WWW_Users.php stop being
> shown.
> Nov 13 commit 2
>
> ***@***.***:/usr/local/www/weberp $ sudo -u www git checkout c8bd38a9e8c85928c99174c93279b5c9527e9f56
> Previous HEAD position was 24c679ae7 Correct default time in the ConvertSQLDateTime() function. Should be 00:00:00 as reported by Pak Ricard
> HEAD is now at c8bd38a9e Images in manual were showing at 32px x 32px size. Correct the css to show at 50% width
> ***@***.***:/usr/local/www/weberp $
>
> FAILS weberp_scc db: WWW_Users.php does NOT show Edit/Delete links for
> user records
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/240#discussioncomment-11279704>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LGMHCZO3655GZLSA7D2A6QD5AVCNFSM6AAAAABR2WFBFCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRXHE3TANA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by pakricard** [comment]
Thanks, @timschofield , now demo is working 

Demo version: is it 4.15.1 as stated on the footer?

**Comment by dalers** [comment]
Imho the demo should be the latest release, i.e. 4.15.2 on PHP 7.

The sooner master branch code stops reporting itself as a release version the better! (again, Imho)

**Comment by pakricard** [comment]
Found the reason of the weird behaviour I find in SecurityTokens.php , a few posts above this one https://github.com/timschofield/webERP/discussions/240#discussioncomment-11274734

Will report it as an issue.

**Comment by timschofield** [comment]
The demo is the latest code from the github master branch. It updates
itself periodically (hourly).

This will mean it sometimes gets broken, but means the latest code is
getting tested more. I calculated this was a price worth paying. Once v5.0
is done then I will make it a stable version.

Thanks
Tim

On Mon, 18 Nov 2024 at 08:36, pakricard ***@***.***> wrote:

> Found the reason of the weird behaviour I find in SecurityTokens.php , a
> few posts above this one #240 (comment)
> <https://github.com/timschofield/webERP/discussions/240#discussioncomment-11274734>
>
> It is due to error_reporting set to -1
>
> How to replicate it:
> Download the standard v 4.15.2 distribution.
> Install it.
> Set error_reporting as -1
> And you will see this error, besides many Notice or Warning messages)
>
> I installed a webERP v4.15.2 as it comes from the zip, on:
> https://ptadu-development.com/TEST/webERP-4.15.2/index.php
> credentials admin/weberp
> Running on PHP 8.2.25 with error_reporting = -1
> where you can see the error.
>
> While I'm doing this job of bringing my webERP to date, I will keep this
> installation (as a benchmark). Fell free to use it if you need.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/240#discussioncomment-11289179>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LFQCSOFZ5EECWB2BLL2BGRJFAVCNFSM6AAAAABR2WFBFCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRYHEYTOOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/
